src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -97,6 +99,33 @@ namespace winrt::TerminalApp::implementation
         _filteredActionsView().SelectionChanged({ this, &CommandPalette::_selectedCommandChanged });
     }
 
+    // Method Description:
+    // - Attempts to load user locale, to further use it when sorting the actions in the filtered view
+    // Return Value:
+    // - The user level locale, upon failure returns a default locale
+    std::locale CommandPalette::_loadUserLocale()
+    {
+        try
+        {
+            // Loading user locale name explicitly using GetUserDefaultLocaleName,
+            // as std:locale("") crashes on some setups.
+            wchar_t userLocaleName[LOCALE_NAME_MAX_LENGTH + 1];
+            auto localeNameSize = GetUserDefaultLocaleName(userLocaleName, ARRAYSIZE(userLocaleName));
```

Comment:
@ShawnSteele, is this an appropriate way of getting the locale for `std::locale` and using it to compare correctly below?